### PR TITLE
Added new example and md file

### PR DESCRIPTION
**Pull request by debajitdatta2k**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1119 @@
+{
```
Comment (telamonian) [inline package-lock.json:1]: Delete this `package-lock.json` file. In the future, only use `yarn` when interacting with the `regular-table` codebase. Never use `npm`

File: package.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
         "clean-css": "^4.2.3",
         "codedown": "^2.1.6",
         "commander": "^5.1.0",
-        "eslint": "^7.1.0",
+        "eslint": "^7.2.0",
```
Comment (telamonian) [inline package.json:55]: This version bump to eslint is off topic for the scope of the current PR. Please revert it, and then once you've done that please also revert the changes you've made to `yarn.lock`

**Comment by telamonian** [comment]
Please don't close your PRs and then open new ones. It makes it very difficult to keep track of any conversations/progress

**Comment by debajitdatta2k** [comment]
> I fully support the idea of getting in a 3D regular-table example. I think this PR has a lot of potential.
> 
> That being said, your two examples are very similar. Also, I just tried to build your `examples/3d_array.md` example, and it's currently quite broken (you seem to be missing a bunch of `` ```javascript `` tags).
> 
> For now, let's just focus on getting your `examples/multidimensional_array.html` example cleaned up and pulled in. Can you please remove all other files/changes from this PR, leaving only the new `examples/multidimensional_array.html` file?

The other files have been restored. 'md' file is removed and examples/multidimensional_array.html has been added.

File: examples/multidimensional_array.html [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    ],
+];
+
+// Global variables declared, to be used later
+
+var num_table; // Current table-dimension number
+const tables = document.getElementsByTagName("tables")[0]; // Select the tag tables
+
+function dataListener(x0, y0, x1, y1) {
+            return {
```
Comment (telamonian) [inline examples/multidimensional_array.html:131]: Be consistent with your leading whitespace. Use 4 spaces per indentation level here and elsewhere

File: examples/multidimensional_array.html [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+                num_rows: DATA[num_table][0].length,
+                num_columns: DATA[num_table].length,
+                data: DATA[num_table].slice(x0, x1).map((col) => col.slice(y0, y1)),
+    };
+}
+
+// The createTables() function takes the 3D array data
+// and displays individual 2D array of individual dimensions
+
+function createTables() {
```
Comment (telamonian) [inline examples/multidimensional_array.html:141]: I think using regular-table to present high dimensional data in an interactive way is neat, and I really like the idea. However, your `createTables` function here is not the best way to go about it.

Even though your underlying data model is 3D, the regular-tables you create here in `createTables` are just 2D. Especially since there isn't any relationship between the slices of `DATA`, just stacking a bunch of tables in a column doesn't make for the most interesting example here.

I think we can do better.

File: examples/multidimensional_array.html [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+
+<body>
+
+    <tables></tables>
+
+    <script>
+        // A random 3-dimensional data is created for
+        // implementing the regular-table concept for 3D array
+
+const DATA = [
```
Comment (telamonian) [inline examples/multidimensional_array.html:52]: `DATA` here is too big, in that it's 60 lines of meaningless static data, and too small, in that 15 x 15 slices are not a good way to show off the capabilities of regular-table.

Part of the appeal of regular-table is virtualization, which means that it can easily be used to render extremely large data sets, such as in the two billion rows example: https://bl.ocks.org/texodus/483a42e7b877043714e18bea6872b039. We should do something more along those lines here

**Comment by telamonian** [comment]
@debajitdatta2k To get you started, I've pushed a commit to this branch that implements the suggestions I made in my [review above](https://github.com/jpmorganchase/regular-table/pull/61#pullrequestreview-434988909). The example now renders a single 3D regular-table that you can scroll "into" by holding down the shift button. The data model is just the `i, j, k` index triplet for each cell, and each cell is colored by the corresponding RGB triplet:


regular_table_3d_rgb_triplets_example


The idea is for this commit to serve as a jumping off point for the initial work on the 3D example. You are welcome to play around with it and make any additions or changes that you think would improve the example. Also, if you have any questions, please feel free to ask

**Comment by texodus** [comment]
Thanks for the PR @debajitdatta2k and the refactor/cleanup/code-review @telamonian!
